.drone.yml

```diff
@@ -1,10 +1,48 @@
+---
 kind: pipeline
 type: docker
-name: build
+name: build-pr
 
 clone:
   disable: true
 
+steps:
+  - name: clone
+    image: plugins/git
+    settings:
+      recursive: true
+      tags: true
+  - name: build
+    image: plugins/docker
+    depends_on: [ clone ]
+    trigger:
+      event:
+        - pull_request
+    settings:
+      repo: hashbang/book
+      username: hashbangbot
+      password:
+        from_secret: dockerhub_ro_passphrase
+      purge: true
+      dry_run: true
+
+trigger:
+  event:
+    - pull_request
+---
```

Comment:
maybe add blank line between sections?

Reply:
When I run the sign it doesn't put a blank line before that second. So I'll assume that most tooling would probably strip the newline